src/test/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceTest.java

```diff
@@ -1211,38 +1136,8 @@ void updateHearingRequestShouldThrowErrorWhenRelatedPartyDetailsAreNotPresent()
             assertEquals(INVALID_RELATED_PARTY_DETAILS, exception.getMessage());
         }
 
-        @Test
-        void updateHearingRequestShouldThrowErrorWhenHearingIdIsNull() {
-            UpdateHearingRequest updateHearingRequest = TestingUtil.updateHearingRequest();
-            Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
-                .updateHearingRequest(null, updateHearingRequest));
-            assertEquals(INVALID_HEARING_ID_DETAILS, exception.getMessage());
-        }
-
-        @Test
-        void updateHearingRequestShouldThrowErrorWhenHearingIdExceedsMaxLength() {
-            UpdateHearingRequest updateHearingRequest = TestingUtil.updateHearingRequest();
-            Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
-                .updateHearingRequest(20000000001111L, updateHearingRequest));
-            assertEquals(INVALID_HEARING_ID_DETAILS, exception.getMessage());
-        }
 
-        @Test
-        void updateHearingRequestShouldThrowErrorWhenHearingIdDoesNotStartWith2() {
-            UpdateHearingRequest updateHearingRequest = TestingUtil.updateHearingRequest();
-            Exception exception = assertThrows(BadRequestException.class, () -> hearingManagementService
-                .updateHearingRequest(1000000100L, updateHearingRequest));
-            assertEquals(INVALID_HEARING_ID_DETAILS, exception.getMessage());
-        }
 
-        @Test
-        void updateHearingRequestShouldThrowErrorWhenHearingIdNotPresentInDB() {
-            when(hearingRepository.existsById(2000000000L)).thenReturn(false);
-            UpdateHearingRequest updateHearingRequest = TestingUtil.updateHearingRequest();
-            Exception exception = assertThrows(HearingNotFoundException.class, () -> hearingManagementService
-                .updateHearingRequest(2000000000L, updateHearingRequest));
-            assertEquals("No hearing found for reference: 2000000000", exception.getMessage());
-        }
```

Comment:
Don't think this is related to your changes- why has this been deleted?

Reply:
Retrieved